article/app/views/liveblog/liveBlogBodyContent.scala.html

```diff
@@ -21,7 +22,23 @@
             _root_.liveblog.LatestBlock(article.blocks)
         }" data-test-id="live-blog-blocks"
         itemprop="@if(article.tags.isReview) {reviewBody} else {articleBody}">
-        @views.html.liveblog.liveBlogBlocks(model.currentPage.currentPage.blocks, article, timezone, amp)
+            @if(amp) {
```

Comment:
would it be clearer to separate the amp case out into its own function, so you can see the if and the else more easily?  doesn't necessarily have to be its own file